### ISSUE TITLE
Rails tagged logger

### ISSUE THREAD

**Issue by arr-dev** [wvanbergen/request-log-analyzer#143]
We are using rails tagged logger to log Doorkeeper OAuth token information.

``` ruby
# config/application.rb
config.log_tags =.[
  :uuid,
  lambda do
   # get token somehow
    "#{token.id}-#{token.application.id}"
  end
]
```

Example log output line:
`[4c7b23acf92ecf0e6be95cb9053dbe15] [9-3]`

Format is setup as per wiki instructions:

``` ruby
class MyCustomLog < RequestLogAnalyzer::FileFormat::Rails3
  line_definition :log_tags do |line|
    line.regexp = /^\[\w*\] \[(\d*)\-(\d*)\]\s*$/

    line.captures <<
      { :name => :token, :type => :integer } <<
      { :name => :app, :type => :integer } 
  end

  report(:append) do |analyze|
    analyze.frequency :token, :title => "OAuth Token", :line_type => :log_tags
    analyze.frequency :app, :title => "OAuth Application", :line_type => :log_tags
  end
end
```

Still in the output I get "None found." for both of my analizers.

I do get "Parsable lines were encountered without a header line before it." error with this regex, but even if I use some other line, like "Completed" I don't get warnings, but also "None found".

I've checked regex with rubular, even tried hardcoding the values, but without luck.

Can someone nudge me in the right direction?
